src/main/kotlin/com/salesforce/dynamodbv2/mt/backups/MtBackupAwsAdaptor.kt

```diff
@@ -0,0 +1,46 @@
+/* Copyright (c) 2019, Salesforce.com, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause.
+ * For full license text, see LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause.
+ */
+package com.salesforce.dynamodbv2.mt.backups
+
+import com.amazonaws.services.dynamodbv2.model.BackupDescription
+import com.amazonaws.services.dynamodbv2.model.BackupDetails
+import com.amazonaws.services.dynamodbv2.model.BackupStatus
+import com.amazonaws.services.dynamodbv2.model.BackupSummary
+import com.amazonaws.services.dynamodbv2.model.DescribeBackupResult
+import java.util.*
+
+/**
+ * Convert simple pojos defined in {@link MtBackupManager} to and from AWS objects defined in AWS specs.
+ */
+class MtBackupAwsAdaptor {
```

Comment:
Why is this needed?

Reply:
not strictly necessary, sort of a utility class for easy unit testing